clients/web/src/lib/components/molecules/user-menu/UserMenu.svelte

```diff
@@ -13,6 +14,7 @@
 
 {#if $user}
 	<div class="flex items-center gap-8">
+		<span> {$currentUser?.data?.me.members[0].players[0].skillGroup.profile.description}</span>
```

Comment:
Do we want to use `members[0].players[0]`? This may mess up if there are more than one members per user or different games per member. May not be an issue for mvp depending on prod database setup. @0c370t

Reply:
[9cd9653](https://github.com/SprocketBot/sprocket/pull/207/commits/9cd9653d885bf5dcb7e34dbf829e804a09bcff3b)